Dockerfile

```
@@ -0,0 +1,19 @@
+FROM python:latest as base
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH=$PATH:/root/.local/bin/
+WORKDIR /app
+COPY ./poetry.toml /app
+COPY ./pyproject.toml /app
```

Very minor, but if you're interested worth noting that we can combine these:
```suggestion
COPY ./poetry.toml ./pyproject.toml /app/
```